### ISSUE TITLE
Exclude MoCo Series from compliance evaluations

### ISSUE THREAD

**Issue by sinhaharsh** [Open-Minds-Lab/mrQA#43]
Siemens automatically generates motion corrected series for fMRI sequences. We observe that several fMRI sequences are collected together under a new sequence, called mocoseries. As they are different sequences with different protocols, it shows up in the report as non-compliance. We saw that the reference protocol computed is very similar to fmri-ringrewards.

How can we differentiate between raw formats and derivatives? Is there a DICOM flag?

**Comment by sinhaharsh** [comment]
> 'Private_0051_1016' means (0051, 1016). This tag should contain a string including MOCO, if the data are postprocessed by the online motion correction.

[Source](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A2=spm;4b96cb15.0905)

**Comment by raamana** [comment]
i think it would be good to have a config file, letting users make different types of rules, inclusions, exclusions, thresholds etc

**Comment by sinhaharsh** [comment]
Right! Can be addressed together with Issue #17

**Comment by sinhaharsh** [comment]
Proposed JSON file template. I think we can move some of the CLI arguments which are never used, but still are useful such as include_phantom, include_nifti_header, skip_parameters

```json
{
  "begin": "2014-03-12T13:37:27+00:00",
  "end": "2017-03-12T13:37:27+00:00",
  "include_sequence": {
    "localizer": false,
    "nifti_header": false,
    "moco": false,
    "sbref": false
  },
  "skip_parameters": [
    "BodyPartExamined",
    "StationName"
  ],
  "exclude_subects": [
    "210098",
    "210078"
  ]
}
```

**Comment by sinhaharsh** [comment]
Can be addressed together with Issue #29
